Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebResultView.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+enum Status {
```

Comment:
We may be able to extract/reuse this across the other features as well - left here for now but I expect many of the other features may have a similar status. If not we can rename this to `PayPalWebStatus`

Reply:
Does this represent the status of the orderID? Maybe we can make the name more descriptive

Reply:
Agreed - renamed to `OrderStatus` here to be more descriptive: https://github.com/paypal/paypal-ios/pull/223/commits/6c304140b0fa866b668711c44eb6c7411b5a437a